lib/ansible/modules/cloud/vmware/vmware_dns_config.py

```diff
@@ -20,106 +20,300 @@
 version_added: 2.0
 author:
 - Joseph Callen (@jcpowermac)
+- Ryan Butler (@ryan_c_butler)
```

Comment:
Would it be possible for you to work on https://github.com/ansible/ansible/pull/47271 which tries to deprecated `vmware_dns_config` and create a new module called `vmware_host_dns` ? Let me know ? Thanks.

Reply:
I guess I should have looked at the open PRs :) Let me know how I can help.

Reply:
Thanks, you can create a new PR using the code available in #47271, test it and let me know.